tests/bearlib/languages/documentation/documentation_extraction_testdata/default.py

```diff
@@ -9,12 +9,8 @@ def foobar_explosion(radius):
     """
     A nice and neat way of documenting code.
     :param radius: The explosion radius. """
-
-
     def get_55():
         """A function that returns 55."""
-
-
         return 55
     return get_55() * radius
```

Comment:
now the fun part ..

```bash
$ pycodestyle default.py
default.py.1:17:1: E305 expected 2 blank lines after class or function definition, found 1
...
```

that is part of this PR.  this is undisputed territory that all module level def & classes have two whitespace before and after them.

Also, there should be a blank line after `def get_55` is finished, before `return get_55() * radius` .  This is a greyish area; I have seen people argue for and against this.  But it becomes clearer (and I am pretty sure Guido comment on this specific case) when there are two inner def's inside a def ; they need to be separated by a blank line.